help/Test_mycbr_py_api.ipynb

```diff
@@ -0,0 +1,689 @@
+{
```

Comment:
Please, clear outputs from ipnb files.

Reply:
This file is work in progress. I was capturing my daily activity. I'll do it once it is completed.

Reply:
Fixed: commit 2008500